src/usdb_syncer/gui/forms/CommentDialog.ui

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+	<class>Dialog</class>
+	<widget class="QDialog" name="Dialog">
+		<property name="geometry">
+			<rect>
+				<x>0</x>
+				<y>0</y>
+				<width>294</width>
+				<height>214</height>
+			</rect>
+		</property>
+		<property name="windowTitle">
+			<string>Post a Comment</string>
+		</property>
+		<layout class="QVBoxLayout" name="verticalLayout">
+			<item>
+				<layout class="QFormLayout" name="formLayout">
+					<item row="0" column="0">
+						<widget class="QLabel" name="label">
+							<property name="text">
+								<string>Comment:</string>
+							</property>
+						</widget>
+					</item>
+					<item row="0" column="1">
+						<widget class="QLineEdit" name="line_edit_username">
```

Comment:
Might be nicer to have a text area instead of a single line.

Reply:
Gotta find the widget for this, and it seems i forgot to alter the name aswell

Would the class be a `QTextEdit`? Or is it another class?

Reply:
There's also QPlainTextEdit, I'm not sure what's appropriate here.

Reply:
Changed it to QPlainTextEdit since we're not sending formated text as far as i know (html/markdown etc)

Reply:
USDB supports HTML in comments, so theoretically it could be formatted, not sure if we want that.

Reply:
Up to you guys if we want it supported (it's easy enough to add in)

Reply:
I'd say leave it out for now, it can be extended later if it is really needed (and users can manually use HTML code as comments if it is really needed).